src/main/bx/components/Grid.bx

```diff
@@ -226,14 +440,35 @@ class{
 			}
 
 			headContent &= "<th class=""bx-grid-column-header#sortClass#""";
-			headContent &= " data-column=""#encodeForHTMLAttribute( column.name )#""";
+			headContent &= " data-column=""#htmlEditFormat( column.name )#""";
 
+			// Build column header styles
+			var headerStyles = [];
 			if( !isNull( column.width ) && len( column.width ) ){
-				headContent &= " style=""width: #column.width#""";
+				headerStyles.append( "width: #column.width#" );
+			}
+			if( !isNull( attributes.colHeaderTextColor ) && len( attributes.colHeaderTextColor ) ){
+				headerStyles.append( "color: #attributes.colHeaderTextColor#" );
+			}
+			if( !isNull( attributes.colHeaderFont ) && len( attributes.colHeaderFont ) ){
+				headerStyles.append( "font-family: #attributes.colHeaderFont#" );
+			}
+			if( !isNull( attributes.colHeaderFontSize ) && len( attributes.colHeaderFontSize ) ){
+				headerStyles.append( "font-size: #attributes.colHeaderFontSize#px" );
+			}
+			if( attributes.colHeaderBold ){
+				headerStyles.append( "font-weight: bold" );
+			}
+			if( attributes.colHeaderItalic ){
+				headerStyles.append( "font-style: italic" );
+			}
+			
+			if( headerStyles.len() ){
+				headContent &= " style=""#headerStyles.toList( "; " )#""";
 			}
 
 			headContent &= ">";
-			headContent &= encodeForHTML( column.header ?: column.name );
```

Comment:
revert all usage of changing `encodeForHTML` to `htmlEditFormat` per the comment above